### ISSUE TITLE
Vector arrow

### ISSUE THREAD

**Issue by kangpyo** [JuliaImages/ImageTracking.jl#33]
Hi, 
This is very useful algorithm for my research. 
Is there a function that show vector arrow instead of color map? 

Thank you

**Comment by johnnychen94** [comment]
Is this https://goropikari.github.io/PlotsGallery.jl/src/quiver.html that you want?

**Comment by kangpyo** [comment]
> Is this https://goropikari.github.io/PlotsGallery.jl/src/quiver.html that you want?

Yes, exactly, is it possible to implement it to your algorithm? I'm particularly using the farneback method.
